doc/underpinnings.tex

```diff
@@ -0,0 +1,779 @@
+      t_{s_1,n_j,ay_k} & t_{s_2,n_n,ay_k} & \dots & t_{s_i,n_j,ay_k} \\
+    };
+
+    \matrix (mB) [draw,matrix of math nodes] at ($(mA.south west)+(1.5,0.7)$)
+    {
+      t_{s_1,n_1,ay_2} & t_{s_2,n_1,ay_2} & \dots & t_{s_i,n_1,ay_2} \\
+      t_{s_1,n_2,ay_2} & t_{s_2,n_2,ay_2} & \dots & t_{s_i,n_2,ay_2} \\
+      \vdots & \vdots & \ddots & \vdots \\
+      t_{s_1,n_j,ay_2} & t_{s_2,n_n,ay_2} & \dots & t_{s_i,n_j,ay_2} \\
+    };
+
+    \matrix (mC) [draw,matrix of math nodes] at ($(mB.south west)+(1.5,0.7)$)
+    {
+      t_{s_1,n_1,ay_1} & t_{s_2,n_1,ay_1} & \dots & t_{s_i,n_1,ay_1} \\
+      t_{s_1,n_2,ay_1} & t_{s_2,n_2,ay_1} & \dots & t_{s_i,n_2,ay_1} \\
+      \vdots & \vdots & \ddots & \vdots \\
+      t_{s_1,n_j,ay_1} & t_{s_2,n_n,ay_1} & \dots & t_{s_i,n_j,ay_1} \\
+    };
+
+    \draw[dashed](mA.north east)--(mC.north east);
+    \draw[dashed](mA.north west)-- node[sloped,above] {Academic Year} (mC.north west);
+    \draw[dashed](mA.south east)--(mC.south east);
+
+    \node [above left] at (mC.north) {Setting};
+    \node [right] at (mC.east) {Need};
+    \node [left] at (mC.west) {$Y_n$};
+
+    \matrix (mD) [draw,matrix of math nodes] at ($(mA.south east)+(0,-3.5)$)
+    {
+      t_{s_1,n_1,ay_k} & t_{s_2,n_1,ay_k} & \dots & t_{s_i,n_1,ay_k} \\
+      t_{s_1,n_2,ay_k} & t_{s_2,n_2,ay_k} & \dots & t_{s_i,n_2,ay_k} \\
+      \vdots & \vdots & \ddots & \vdots \\
+      t_{s_1,n_j,ay_k} & t_{s_2,n_n,ay_k} & \dots & t_{s_i,n_j,ay_k} \\
+    };
+
+    \matrix (mE) [draw,matrix of math nodes] at ($(mB.south east)+(0,-3.5)$)
+    {
+      t_{s_1,n_1,ay_2} & t_{s_2,n_1,ay_2} & \dots & t_{s_i,n_1,ay_2} \\
+      t_{s_1,n_2,ay_2} & t_{s_2,n_2,ay_2} & \dots & t_{s_i,n_2,ay_2} \\
+      \vdots & \vdots & \ddots & \vdots \\
+      t_{s_1,n_j,ay_2} & t_{s_2,n_n,ay_2} & \dots & t_{s_i,n_j,ay_2} \\
+    };
+
+    \matrix (mF) [draw,matrix of math nodes] at ($(mC.south east) + (0,-3.5)$)
+    {
+      t_{s_1,n_1,ay_1} & t_{s_2,n_1,ay_1} & \dots & t_{s_i,n_1,ay_1} \\
+      t_{s_1,n_2,ay_1} & t_{s_2,n_2,ay_1} & \dots & t_{s_i,n_2,ay_1} \\
+      \vdots & \vdots & \ddots & \vdots \\
+      t_{s_1,n_j,ay_1} & t_{s_2,n_n,ay_1} & \dots & t_{s_i,n_j,ay_1} \\
+    };
+
+    \draw[dashed](mD.north east)--(mF.north east);
+    \draw[dashed](mD.north west)--(mF.north west);
+    \draw[dashed](mD.south east)--(mF.south east);
+
+    \draw[dotted,->](mC.south)--(mE.north);
+    \draw[dotted,->](mB.south)--(mD.north);
+
+    \node [left] at (mF.west) {$Y_{n+1}$};
+  \end{tikzpicture}
+  \caption{State Transitions in the Send Markov Chain\label{state}
+  }
+\end{figure}
+
+\subsection{Markov Chain}
+
+This section and the next are a detour into how a the Send model would
+look if it were implemented using just Markov Chains.  If we
+understand this we understand why we can see the Send Model is''
+`Markov like`''.
+
+The initial state of our Markov Chain, $Y_0$ is constructed from the
+historic data provided by the client (we simply count the number of
```

Comment:
reword section in brackets:
(we simply count the number of
pupils that, for example, match $(s_1,n_1,ay_1)$)